.github/workflows/test-run-tails.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
         shell: bash -l {0}
         run: |
           python -m pip install --upgrade pip
-          pip install wheel==0.36.0
+          pip install "wheel>=0.36.0" "pyparsing>=2.4,<3"
```

Comment:
Why pin to an old version of pyparsing?

Reply:
This is because `httplib2` has that in its deps, see here: https://github.com/httplib2/httplib2/network/dependencies (scroll to the bottom); pip tries to install the latest `3.0.0b2`, which then causes a conflict that it can't resolve. Seems like a fairly recent thing, didn't see that the last time the CI was run (4 weeks ago).